src/Orleans/Serialization/FieldUtils.cs

```diff
@@ -0,0 +1,110 @@
+using System;
+using System.Reflection;
+using System.Reflection.Emit;
+
+namespace Orleans.Serialization
+{
+    public sealed class FieldUtils
```

Comment:
This class will eventually move to Abstractions? If that's the case, and I'm not suggesting this for the beta but for longer term, can we avoid having it in Abstractions and instead have the codegen add this method to the generated code if it's required?

Reply:
We can keep that in mind